great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -943,26 +943,13 @@ def _persist_suite_with_store(
     @override
     def _save_project_config(
         self, _fds_datasource: FluentDatasource | None = None
-    ) -> FluentDatasource | None:
+    ) -> None:
         """
         See parent 'AbstractDataContext._save_project_config()` for more information.
 
         Explicitly override base class implementation to retain legacy behavior.
         """
-        # 042723 kilo59
-        # Currently CloudDataContext and FileDataContext diverge in how FDS are persisted.
-        # FileDataContexts don't use the DatasourceStore at all to save or hydrate FDS configs.
-        # CloudDataContext does use DatasourceStore in order to make use of the Cloud http clients.
-        # The intended future state is for a new FluentDatasourceStore that can fully encapsulate
-        # the different requirements for FDS vs BDS.
-        # At which time `_save_project_config` will revert to being a no-op operation on the CloudDataContext.
-        if _fds_datasource:
-            return self._datasource_store.set(key=None, value=_fds_datasource)
-        else:
-            logger.debug(
-                "CloudDataContext._save_project_config() has no `fds_datasource` to update"
-            )
-            return None
+        return None
```

Comment:
Could we log here instead of completely no-oping?

Reply:
Sure. Addressed in https://github.com/great-expectations/great_expectations/pull/8773/commits/460c41b8e2a7becf6e97d41b97d1ba61875d0225.